wasm/src/logic/mod.rs

```diff
@@ -45,6 +45,41 @@ impl LogicContext {
             self.handle_message(msg)?;
         }
 
+        // TODO: Remove this temporary sprite loading. Replace it with some kind of
+        // "resource complete" event
+        {
+            use rask_engine::resources::GetStore;
+            let res = crate::communication::RESOURCE_TABLE.read();
+            let tex1: Result<&rask_engine::resources::Texture, _> = res.get(0);
```

Comment:
Implement Resourcetabel.get(ResourceInfo) to avoid magic numbers

Reply:
Yeah, of cause, I'll fix that